Disable the update function so it doesn't break the package.
--- a/searchsploit
+++ b/searchsploit
@@ -48,7 +48,7 @@ function usage()
   echo "   -c, --case     Perform a case-sensitive search (Default is insensitive)."
   echo "   -f, --file     Include file's path when searching (Default is just the exploit title)."
   echo "   -h, --help     Show this help screen."
-  echo "   -u, --update   Update exploit database from git."
+#  echo "   -u, --update   Update exploit database from git."
   echo "   -v, --verbose  Verbose output. Title lines are allowed to overflow their columns."
   echo "   -w, --www      Show URLs to Exploit-DB.com rather than local path."
   echo "       --colour   Disable colour highlighting."
@@ -94,7 +94,7 @@ for param in "$@"; do
   elif [[ "${param}" == "--case" ]]; then
     SCASE=''
   elif [[ "${param}" == "--update" ]]; then
-    UPDATE=1
+    UPDATE=0
   elif [[ "${param}" == "--verbose" ]]; then
     VERBOSE=1
   elif [[ "${param}" == "--id" ]]; then
@@ -122,7 +122,7 @@ while getopts "uchvwf" arg "${ARGS}"; do
   case ${arg} in
     c) SCASE='';;
     h) usage >&2;;
-    u) UPDATE=1;;
+    u) UPDATE=0;;
     v) VERBOSE=1;;
     w) WEBLINK=1;;
     f) FILEPATH=1;;
